proto/config/devconfig.proto

```diff
@@ -169,6 +169,14 @@ message EdgeDevConfig {
 
   // cluster configuration
   EdgeNodeCluster cluster = 43;
+  DhcpVendorExtensionsOverride dhcp_extensions_override = 44;
+}
+
+// DhcpVendorExtensionsOverride is used to override or add dhcp vendor extensions like f.e.
+// the NTP servers - currently no other options are supported, but DNS servers might be a
+// good candidate in the future
+message DhcpVendorExtensionsOverride {
```

Comment:
I think that terminology 'vendor extensions' is more used with BOOTP. For DHCP the [RFC](https://datatracker.ietf.org/doc/html/rfc2132#section-2) just refers to them as Options.
So what about calling this `DhcpOptionsOverride`?

Reply:
I changed it to `DhcpOptionsIgnore` https://github.com/lf-edge/eve-api/pull/70/files#diff-35ea4ed74707f6ad6d17be169425a7acf4e25e09e9de81a0f7300ed80c5ec656R101